manifests/q/QTTabBar/QTTabBar/1.5.260/QTTabBar.QTTabBar.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: QTTabBar.QTTabBar
+PackageVersion: 1.5.260
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: https://telkomuniversity.dl.sourceforge.net/project/qttabbar/1.5.0.0%20Beta%202/qttabbar-v1.5.0.0b2.msi

```

```suggestion
  InstallerUrl: https://sourceforge.net/projects/qttabbar/files/1.5.0.0%20Beta%202/qttabbar-v1.5.0.0b2.msi/download
```
Due to dynamic CDN, the redirected link here should be used to avoid speed issues